akka-stream/src/main/scala/akka/stream/scaladsl/RestartFlow.scala

```diff
@@ -281,6 +282,7 @@ private abstract class RestartWithBackoffLogic[S <: Shape](
         if (finishing || maxRestartsReached() || onlyOnFailures) {
           complete(out)
         } else {
+          logIt(s"Restarting stream due to completion [${restartCount + 1}]", OptionVal.None)
```

Comment:
This case was previously silent, I thought it would be good to log it in similar way. wdyt?

Reply:
I like adding logging for it, but as it could be intended behavior perhaps it should always be on 'info' level? (or configurable separately, but that might be going too far ;) )

Reply:
ok, I'll make it info and only log if logging is enabled (by the log attribute)